pkgs/development/libraries/libvirt/default.nix

```diff
@@ -267,6 +271,7 @@ stdenv.mkDerivation rec {
       (cfg "runstatedir" "/run")
 
       (cfg "init_script" (if isDarwin then "none" else "systemd"))
+      (cfg "qemu_datadir" (if isDarwin then "${qemu}/share/qemu" else ""))
```

Comment:
this will pass `-Dqemu_datadir=` on linux so do

```patch
diff --git a/pkgs/development/libraries/libvirt/default.nix b/pkgs/development/libraries/libvirt/default.nix
index a8f612678d0..1421588523c 100644
--- a/pkgs/development/libraries/libvirt/default.nix
+++ b/pkgs/development/libraries/libvirt/default.nix
@@ -271,7 +271,6 @@ stdenv.mkDerivation rec {
       (cfg "runstatedir" "/run")
 
       (cfg "init_script" (if isDarwin then "none" else "systemd"))
-      (cfg "qemu_datadir" (if isDarwin then "${qemu}/share/qemu" else ""))
 
       (feat "apparmor" isLinux)
       (feat "attr" isLinux)
@@ -330,6 +329,8 @@ stdenv.mkDerivation rec {
       (storage "scsi" true)
       (storage "vstorage" isLinux)
       (storage "zfs" enableZfs)
+    ] ++ lib.optionals isDarwin [
+      (cfg "qemu_datadir" ("${qemu}/share/qemu"))
     ];
 
   doCheck = true;
```

Reply:
or should this perhaps be applied on linux too?

Reply:
or should this perhaps be applied on linux too?

Reply:
I looked at the libvirt source before making the PR -- if the empty string is passed it uses the default value. I'll post a link shortly.

Reply:
@Artturin: https://github.com/libvirt/libvirt/blob/56de80cb793aa7aedc45572f8b6ec3fc32c99309/meson.build#L1645